src/main/scala-3/com/typesafe/sbt/packager/PluginCompat.scala

```diff
@@ -0,0 +1,58 @@
+package com.typesafe.sbt.packager
+
+import java.io.File
+import java.nio.file.{Path => NioPath}
+import java.util.jar.Attributes
+import sbt.*
+import xsbti.{FileConverter, HashedVirtualFileRef, VirtualFile, VirtualFileRef}
+import sbt.internal.RemoteCache
+
+object PluginCompat:
```

Comment:
This is the only file using the whitespace sensitive syntax. Is this the way scala will evolve? I personally am not to fond of whitespace sensitive syntax ( granted, YAML is my main nemesis :stuck_out_tongue_winking_eye:  ). My argument here would be code style consistency.

Reply:
For Scala 3 only code bases, I exclusively use indent syntax, and gradually switch into it whenever I touch the code. I'm guessing that some are like me, and others might be sticking with curly braces?

I can change it to use curly braces if you want.

Reply:
Added curly braces in 2a20c607c0b476feff5d959f06983d243e6f0922